products/jbrowse-web/src/JBrowse.js

```diff
@@ -84,6 +86,27 @@ const JBrowse = observer(({ pluginManager }) => {
     return disposer
   }, [rootModel, setSession, useLocalStorage, useUpdateUrl, session])
 
+  useEffect(() => {
+    onSnapshot(rootModel, async snapshot => {
+      if (adminMode) {
+        const payload = { adminKey: adminKeyParam, config: snapshot.jbrowse }
+        const response = await fetch('/updateConfig', {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(payload),
+        })
+        if (!response.ok) {
+          const message = await response.text()
+          rootModel.session.notify(
+            `got ${response.status} (${response.statusText}) ${message || ''}`,
```

Comment:
Maybe make the error message be a bit more descriptive, so the user knows where the notification is coming from.

```suggestion
            `Admin server error: ${response.status} (${response.statusText}) ${
              message || ''
            }`,
```